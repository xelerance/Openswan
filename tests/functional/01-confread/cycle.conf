version	2.0	# conforms to second version of ipsec.conf specification

include /cycle1.conf



