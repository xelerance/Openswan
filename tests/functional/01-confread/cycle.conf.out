config file: cycle.conf can not be loaded: cycle/cycle1.conf:2: max inclusion depth reached [
]
